src/controllers/simplified-account/settings/email-notifications/email-notifications.controller.js

```diff
@@ -1,8 +1,6 @@
 const { response } = require('../../../../utils/response')
 const formatSimplifiedAccountPathsFor = require('../../../../utils/simplified-account/format/format-simplified-account-paths-for')
-const paths = require('../../../../paths')
-const { setEmailCollectionModeByServiceIdAndAccountType } = require('../../../../services/email.service')
-const logger = require('../../../../utils/logger')(__filename)
+const paths = require('@root/paths')
 
 function getEmailNotificationsSettingsPage (req, res) {
```

Comment:
```suggestion
function get (req, res) {
```
can be just `get`

Reply:
This PR is for email collection mode only. email-notifications.controller.js will be refactored in a separate PR.